

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Broman et al. (2019) Fig 1</title>
    
    
    <meta name="author" content="Chiranjib Saha">

    <!-- Enable responsive viewport -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le styles -->
    <link href="/assets/themes/twitter/bootstrap/css/bootstrap.2.2.2.min.css" rel="stylesheet">
    <link href="/assets/themes/twitter/css/style.css?body=1" rel="stylesheet" type="text/css" media="all">
    <link href="/assets/themes/twitter/css/kbroman.css" rel="stylesheet" type="text/css" media="all">
    <link href = "maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css",rel = "stylesheet">
    <link href = "maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js",rel = "stylesheet">


    <!-- Le fav and touch icons -->
    <link rel="SHORTCUT ICON" href="/assets/me.ico"/>

    <!-- atom & rss feed -->
    <link href="nil" type="application/atom+xml" rel="alternate" title="Sitewide ATOM Feed">
    <link href="nil" type="application/rss+xml" rel="alternate" title="Sitewide RSS Feed">
  
    <style>
      @import url('https://fonts.googleapis.com/css?family=Lato');
      @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700');
      @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700|PT+Sans');
    </style>
  </head>

  <body>
    <div class="navbar">
      <div class="navbar-inner">
        <div class="container-narrow">
          <a class="brand" href="/">Chiranjib Saha</a>
          <ul class="nav">
              <li><a href="/index.html">About</a></li>
              <li><a href="/pages/news.html">News</a></li>
              <li><a href="/pages/pubs.html">Publications</a></li>
              <!--li><a href="/pages/projects.html">Projects</a></li>
              <li><a href="/pages/talks.html">talks</a></li>
              <li><a href="/pages/software.html">software</a></li>
              <li><a href="/pages/teaching.html">teaching</a></li>
              <li><a href="/pages/tutorials.html">tutorials</a></li>-->
          </ul>
        </div>
      </div>
    </div>

    <div class="container-narrow">

      <div class="content">
        

<div class="row-fluid">
  <div class="span12">
    <div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="samplemixups_fig7.html">prev</a></li>
          <li><a href="isletc6_fig4.html">next</a></li>
      </ul>
  </div>
</div>

<p>Broman KW, Gatti DM, Simecek P, Furlotte NA, Prins P, Sen Ś,
Yandell BS, Churchill GA (2019)
R/qtl2: software for mapping quantitative trait loci with
high-dimensional data and multi-parent populations.
<a href="http://genetics.org">Genetics</a> 211:495-502
<a href="https://www.ncbi.nlm.nih.gov/pubmed/30591514"><img src="../icons16/pubmed-icon.png" alt="PubMed" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/publications/rqtl2_paper.pdf"><img src="../icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Paper_Rqtl2"><img src="../icons16/github-icon.png" alt="GitHub" /></a>
<a href="https://kbroman.org/qtl2"><img src="../icons16/R-icon.png" alt="R/qtl2 software" /></a>
<a href="https://doi.org/10.1534/genetics.118.301595"><img src="../icons16/doi-icon.png" alt="doi" /></a></p>

<p><img src="../../assets/bigpublpics/rqtl2_fig1_lg.png" alt="Broman et al. (2019) Fig 1" /></p>

<p><strong>Figure 1</strong>. — Reconstruction of Fig. 5 from <a href="https://doi.org/10.1534/g3.114.013748">Gatti et al.
(2014)</a>, on the mapping of
constitutive neutrophil counts in 742 Diversity Outbred mice. (A) LOD
scores from a genome scan using the full model (comparing all 36
genotypes for the autosomes and 44 genotypes for the X chromosome);
the dashed horizontal line indicates the 5\% genome-wide significance
threshold, based on a permutation test. (B) LOD scores from a genome
scan with an additive allele model (compare the 8 founder haplotypes).
(C) LOD scores from a SNP association scan, using all SNPs that were
genotyped in the eight founder lines. (D) Best linear unbiased
predictors (BLUPs) of the eight haplotype effects in the additive
model, along with the LOD curve on chromosome 1. (E) SNP association
results in the region of the chromosome 1 QTL, along with genes in the
region; SNPs with LOD scores within 1.5 of the maximum are highlighted
in pink. All figures are produced with <a href="https://kbroman.org/qtl2">R/qtl2</a>.</p>

  </div>
</div>


      </div>

      <hr>

    </div>

    
  </body>
</html>

