

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Broman et al. (2015) Fig 7</title>
    
    
    <meta name="author" content="Chiranjib Saha">

    <!-- Enable responsive viewport -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le styles -->
    <link href="/assets/themes/twitter/bootstrap/css/bootstrap.2.2.2.min.css" rel="stylesheet">
    <link href="/assets/themes/twitter/css/style.css?body=1" rel="stylesheet" type="text/css" media="all">
    <link href="/assets/themes/twitter/css/kbroman.css" rel="stylesheet" type="text/css" media="all">
    <link href = "maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css",rel = "stylesheet">
    <link href = "maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js",rel = "stylesheet">


    <!-- Le fav and touch icons -->
    <link rel="SHORTCUT ICON" href="/assets/me.ico"/>

    <!-- atom & rss feed -->
    <link href="nil" type="application/atom+xml" rel="alternate" title="Sitewide ATOM Feed">
    <link href="nil" type="application/rss+xml" rel="alternate" title="Sitewide RSS Feed">
  
    <style>
      @import url('https://fonts.googleapis.com/css?family=Lato');
      @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700');
      @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700|PT+Sans');
    </style>
  </head>

  <body>
    <div class="navbar">
      <div class="navbar-inner">
        <div class="container-narrow">
          <a class="brand" href="/">Chiranjib Saha</a>
          <ul class="nav">
              <li><a href="/index.html">About</a></li>
              <li><a href="/pages/news.html">News</a></li>
              <li><a href="/pages/pubs.html">Publications</a></li>
              <!--li><a href="/pages/projects.html">Projects</a></li>
              <li><a href="/pages/talks.html">talks</a></li>
              <li><a href="/pages/software.html">software</a></li>
              <li><a href="/pages/teaching.html">teaching</a></li>
              <li><a href="/pages/tutorials.html">tutorials</a></li>-->
          </ul>
        </div>
      </div>
    </div>

    <div class="container-narrow">

      <div class="content">
        

<div class="row-fluid">
  <div class="span12">
    <div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="tian2016_fig4.html">prev</a></li>
          <li><a href="rqtl2_fig1.html">next</a></li>
      </ul>
  </div>
</div>

<p>Broman KW, Keller MP, Broman AT, Kendziorski C, Yandell BS, Sen
Ś, Attie AD (2015) Identification and correction of sample
mix-ups in expression genetic data: A case study. <a href="http://g3journal.org">G3</a>
5:2177-2186
<a href="https://www.ncbi.nlm.nih.gov/pubmed/26290572"><img src="../icons16/pubmed-icon.png" alt="PubMed" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/publications/sample_mixups.pdf"><img src="../icons16/pdf-icon.png" alt="pdf" /></a>
<a href="http://bit.ly/B6BTBR"><img src="../icons16/data-icon.png" alt="data" /></a>
<a href="https://github.com/kbroman/Paper_SampleMixups"><img src="../icons16/github-icon.png" alt="GitHub" /></a>
<a href="https://github.com/kbroman/lineup"><img src="../icons16/R-icon.png" alt="R/lineup software" /></a></p>

<p><img src="../../assets/bigpublpics/samplemixups_fig7_lg.png" alt="Broman et al. (2015) Fig 7" /></p>

<p><strong>Figure 7</strong>. — The DNA sample mix-ups on the seven 96-well plates used
for genotyping. Black dots indicate that the correct DNA was put in
the well. Blue arrows point from where a sample should have been
placed to where it was actually placed; the different shades of blue
convey no meaning. Red X’s indicate DNA samples that were omitted.
Orange arrowheads indicate wells with incorrect samples, but the
sample placed there is of unknown origin. Purple and green
arrow-heads indicate cases where the sample placed in the well was
incorrect, but the DNA that was supposed to be there was not found;
with the purple cases, there was corresponding gene expression data,
while for the green cases, there was no corresponding gene
expression data. Pink circles (e.g., well D02 on plate 1631)
indicate sample duplicates. Gray dots indicate that the sample
placed in the well cannot be verified, as there was no corresponding
gene expression data. Gray circles indicate controls or unused
wells.</p>

  </div>
</div>


      </div>

      <hr>

    </div>

    
  </body>
</html>

