

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>software</title>
    <meta name="description" content="Karl Broman's software">
    
    <meta name="author" content="Chiranjib Saha">

    <!-- Enable responsive viewport -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le styles -->
    <link href="/assets/themes/twitter/bootstrap/css/bootstrap.2.2.2.min.css" rel="stylesheet">
    <link href="/assets/themes/twitter/css/style.css?body=1" rel="stylesheet" type="text/css" media="all">
    <link href="/assets/themes/twitter/css/kbroman.css" rel="stylesheet" type="text/css" media="all">
    <link href = "maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css",rel = "stylesheet">
    <link href = "maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js",rel = "stylesheet">


    <!-- Le fav and touch icons -->
    <link rel="SHORTCUT ICON" href="/assets/me.ico"/>

    <!-- atom & rss feed -->
    <link href="nil" type="application/atom+xml" rel="alternate" title="Sitewide ATOM Feed">
    <link href="nil" type="application/rss+xml" rel="alternate" title="Sitewide RSS Feed">
  
    <style>
      @import url('https://fonts.googleapis.com/css?family=Lato');
      @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700');
      @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700|PT+Sans');
    </style>
  </head>

  <body>
    <div class="navbar">
      <div class="navbar-inner">
        <div class="container-narrow">
          <a class="brand" href="/">Chiranjib Saha</a>
          <ul class="nav">
              <li><a href="/index.html">About</a></li>
              <li><a href="/pages/news.html">News</a></li>
              <li><a href="/pages/pubs.html">Publications</a></li>
              <!--li><a href="/pages/projects.html">Projects</a></li>
              <li><a href="/pages/talks.html">talks</a></li>
              <li><a href="/pages/software.html">software</a></li>
              <li><a href="/pages/teaching.html">teaching</a></li>
              <li><a href="/pages/tutorials.html">tutorials</a></li>-->
          </ul>
        </div>
      </div>
    </div>

    <div class="container-narrow">

      <div class="content">
        

<div class="page-header">
  <h2>software </h2>
</div>

<div class="row-fluid">
  <div class="span12">
    <div class="navbar">
    <div class="navbar-inner">
        <ul class="nav">
            <li><a href="https://github.com/kbroman">github</a></li>
            <li><a href="http://rqtl.org">R/qtl</a></li>
            <li><a href="https://kbroman.org/qtlcharts">R/qtlcharts</a></li>
        </ul>
    </div>
</div>

<hr />

<h4 id="rqtl"><a name="qtl"></a><a href="http://rqtl.org">R/qtl</a></h4>

<p>An <a href="https://www.r-project.org/">R</a> package with tools for analyzing QTL experiments.</p>

<p>To receive announcements about updates to R/qtl, join the
Google group, <a href="https://groups.google.com/group/rqtl-announce">R/qtl announcements</a>.</p>

<p>To participate in discussion about the use of R/qtl, join the
Google group, <a href="https://groups.google.com/group/rqtl-disc">R/qtl discussion</a></p>

<hr />

<h4 id="rqtl2"><a name="qtl2"></a><a href="https://kbroman.org/qtl2">R/qtl2</a></h4>

<p>A reimplementation of <a href="http://rqtl.org">R/qtl</a>, to better handle
high-dimensional data and complex cross designs (such as the
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15514660">Collaborative Cross</a>,
<a href="https://www.ncbi.nlm.nih.gov/pubmed/18295532">MAGIC</a>,
<a href="https://www.ncbi.nlm.nih.gov/pubmed/11973314">heterogeneous stock</a>,
and
<a href="https://www.ncbi.nlm.nih.gov/pubmed/2234561">Diversity Outbred mice</a>).</p>

<p>To receive announcements about updates to R/qtl2 (and R/qtl1), join the
Google group, <a href="https://groups.google.com/group/rqtl-announce">R/qtl announcements</a>.</p>

<p>To participate in discussion about the use of R/qtl2, join the
Google group, <a href="https://groups.google.com/group/rqtl2-disc">R/qtl2 discussion</a></p>

<hr />

<h4 id="rqtlcharts-and-d3panels"><a name="qtlcharts"></a><a href="https://kbroman.org/qtlcharts">R/qtlcharts</a> and <a href="https://kbroman.org/d3panels">d3panels</a></h4>

<p><a href="https://kbroman.org/qtlcharts">R/qtlcharts</a> is an
<a href="https://www.r-project.org">R</a> package providing <a href="https://d3js.org">D3</a>-based
interactive graphics for QTL experiments.</p>

<p>It is built on the <a href="https://kbroman.org/d3panels">d3panels</a> library
(written in <a href="https://coffeescript.org/">CoffeeScript</a>), which provides
a set of graphics panels of different types (e.g., scatterplot,
dotchart, and heatmap).</p>

<hr />

<h4 id="d3-examples"><a name="D3"></a><a href="https://www.biostat.wisc.edu/~kbroman/D3/">D3 examples</a></h4>

<p>Various examples of <a href="https://d3js.org">D3</a>-based interactive graphs.</p>

<hr />

<h4 id="rbroman"><a name="broman"></a><a href="https://github.com/kbroman/broman">R/broman</a></h4>

<p>Various R functions that I find useful, assembled into an R package.</p>

<hr />

<h4 id="arxiv-and-ieeer"><a name="aRxiv"></a><a href="https://github.com/ropensci/aRxiv">aRxiv</a> and <a name="IEEER"></a><a href="https://github.com/ropensci/IEEER">IEEER</a></h4>

<p><a href="https://www.r-project.org">R</a> packages for searching
<a href="https://arxiv.org">arXiv</a> (an archive of electronic preprints for
computer science, mathematics, physics, quantitative biology,
quantitative finance, and statistics) and
<a href="https://ieeexplore.ieee.org/Xplore/home.jsp">IEEE journals</a>, respectively.
These packages are part of the
<a href="https://ropensci.org">rOpenSci project</a>.</p>

<hr />

<h4 id="rlineup"><a name="lineup"></a><a href="https://github.com/kbroman/lineup">R/lineup</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for detecting and correcting
sample mix-ups between two sets of measurements, such as between gene
expression data on two tissues. (Also see
<a href="https://github.com/kbroman/lineup2">R/lineup2</a>, revised to be more
general and not so closely tied to the <a href="http://rqtl.org">R/qtl</a>
package.)</p>

<hr />

<h4 id="rsimcross"><a name="simcross"></a><a href="https://kbroman.org/simcross">R/simcross</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for simulating general
experimental crosses (such as advanced intercross lines and the
diversity outcross), for use with <a href="http://rqtl.org">R/qtl</a>.</p>

<hr />

<h4 id="rxoi"><a name="xoi"></a><a href="https://github.com/kbroman/xoi">R/xoi</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for the analysis of crossover
interference in experimental crosses, particularly regarding the gamma
model.</p>

<hr />

<h4 id="rlmmlite"><a name="lmmlite"></a><a href="https://kbroman.org/lmmlite">R/lmmlite</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for the fit of a simple
linear mixed model useful for genome-wide association studies
(GWAS) and quantitative trait locus (QTL) mapping.</p>

<hr />

<h4 id="rricalc"><a name="ricalc"></a><a href="https://github.com/kbroman/ricalc">R/ricalc</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for calculation of two- and three-locus probabilities in
multiple-strain recombinant inbred lines, and the simulation of such lines.
See the related paper: Broman KW (2005) The genomes of
recombinant inbred lines.  <a href="http://www.genetics.org/">Genetics</a> 169:1133-1146
[<a href="https://www.ncbi.nlm.nih.gov/pubmed/15545647">PubMed</a> | <a href="https://www.biostat.wisc.edu/kbroman/~publications/rigenome.pdf">pdf (525k)</a>]</p>

<hr />

<h4 id="rmbmixture"><a name="mbmixture"></a><a href="https://github.com/kbroman/mbmixture">R/mbmixture</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for evaluating whether a microbiome sample is the mixture of
two source samples. We are thinking of shotgun sequencing data on the
microbiome sample plus dense SNP genotype data on the two potential
source samples. We assume that the data has been reduced to a
three-dimensional array of read counts: the 3 possible SNP genotypes
for the first sample × the 3 possible SNP genotypes of the
second sample × the 2 possible SNP alleles on the reads. We then
fit a model with contaminant probability <em>p</em> = proportion of the
microbiome sample coming from the second sample and <em>e</em> = rate of sequencing
errors.</p>

<hr />

<h4 id="rnegenes"><a name="negenes"></a><a href="https://github.com/kbroman/negenes">R/negenes</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for estimating the number of essential genes in a genome
on the basis of data from a random transposon mutagenesis experiment.
See the related <a href="https://www.biostat.wisc.edu/~kbroman/publications/ms0220.pdf">technical report</a>.</p>

<hr />

<h4 id="rnpem"><a name="npem"></a><a href="https://github.com/kbroman/npem">R/npem</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for analyzing cell proliferation assays using the
normal-Poisson mixture model described by Broman et al.
(<a href="https://www.ncbi.nlm.nih.gov/pubmed/8946008">J Immunol Meth 198:119-132, 1996</a>).</p>

<hr />

<h4 id="rfingers"><a name="fingers"></a><a href="https://github.com/kbroman/fingers">R/fingers</a></h4>

<p>An <a href="https://www.r-project.org">R</a> package for the inference of full sibling families with data on
dominant genetic markers (such as RAPDs), following the hierarchicical
clustering approach of Apostol et al. (1993) Theor Appl Genet
86:991-1000.  The title follows from Fortran software written by WC
Black IV.</p>

<hr />

<h4 id="rgeesibsor"><a name="geesibsor"></a><a href="https://github.com/kbroman/geesibsor">R/geesibsor</a></h4>

<p>Software for estimating the odds ratio for siblings for a binary
phenotype, adjusting for covariates, using generalized estimating equations.</p>

<hr />

<h4 id="rqtlsim"><a name="qtlsim"></a><a href="https://github.com/kbroman/qtlsim">R/qtlsim</a></h4>

<p>Software for testing, by computer simulation, various approaches
to mapping QTLs in a backcross experiment.  This was written for the
manuscript Broman and Speed (2002) <a href="https://www.biostat.wisc.edu/~kbroman/publications/index.html#rss">A model
selection approach for the identication of quantitative trait loci in
experimental crosses</a>. J Roy Stat Soc B
64:641-656.  Also see the <a href="http://rqtl.org">R/qtl
package</a>.</p>

<hr />

<h4 id="relcheck"><a name="relcheck"></a>RelCheck</h4>
<p>Version 0.67 (24 August 2000)
Software for verifying relationships between all pairs of
individuals in a linkage study, using the approach of Boehnke and Cox
(<a href="https://www.ncbi.nlm.nih.gov/pubmed/9311748">Am
J Hum Genet, 61:423-429, 1997</a>), with the modification described by
Broman and Weber (<a href="https://www.ncbi.nlm.nih.gov/pubmed/9792888">Am
J Hum Genet 63:1563-1564, 1998</a>), to allow for the presence of
genotyping errors.  We look only at the relationships MZ twins,
parent/offspring, fullsibs, halfsibs and unrelated.</p>

<p><strong>Note</strong>: I am no longer actively developing this software. You may wish to check out Mary Sara McPeek and Lei
Sun’s program <a href="http://galton.uchicago.edu/~mcpeek/software/prest/">PREST</a>.  It has a similar aim, and calculates a
more extensive set of statistics, includes measures of statistical
significance, and also looks at avuncular and first cousin
relationships.  Other alternatives include the programs Borel,
Relpair, Relative, Reltype and Siberror.  Go to the
<a href="http://www.jurgott.org/linkage/ListSoftware.html">Rockefeller software list</a>
to find these programs.</p>

<p>The input/output for my program is rather crude.  If you use the
software, please reference the above papers in any publications.</p>

<p>A perl script for converting data from linkage format to that used by
RelCheck is included with the software.</p>

<p>Download:
<a href="https://www.biostat.wisc.edu/software/relcheck/relcheck_0.67.tar.gz">source</a> | <a href="https://www.biostat.wisc.edu/software/relcheck/relcheck_0.67.zip">windows</a></p>

<p>Sample data [<a href="https://www.biostat.wisc.edu/software/relcheck/sampledata.tar.gz">tar.gz</a> |
<a href="https://www.biostat.wisc.edu/software/relcheck/sampledata.zip">zip</a>]
README file: [<a href="https://www.biostat.wisc.edu/software/relcheck/README.txt">README.txt</a>]
List of updates to the software: [<a href="https://www.biostat.wisc.edu/software/relcheck/CHANGES.txt">CHANGES.txt</a>]</p>

<hr />

<h4 id="f2"><a name="f2"></a>f2</h4>

<p>Version 0.50 (7 Feb 2000)</p>

<p>Software for QTL analysis of an F2 intercross experiment,
including forward selection for multiple QTLs, all pairs of loci, and
pairwise interactions.</p>

<p><strong>Note</strong>: This is very preliminary, the input and output
are not well documented, and I’m no longer actively developing this software.  Look at <a href="http://rqtl.org">R/qtl</a>, instead.</p>

<table>
  <tbody>
    <tr>
      <td>Download: <a href="https://www.biostat.wisc.edu/software/f2/f2_0.50.tar.gz">source</a></td>
      <td><a href="https://www.biostat.wisc.edu/software/f2/f2_0.50.zip">windows</a></td>
    </tr>
  </tbody>
</table>

<table>
  <tbody>
    <tr>
      <td>Sample data [<a href="https://www.biostat.wisc.edu/software/f2/example.tar.gz">tar.gz</a></td>
      <td><a href="https://www.biostat.wisc.edu/software/f2/example.zip">zip</a>]</td>
    </tr>
  </tbody>
</table>

  </div>
</div>


      </div>

      <hr>

    </div>

    
  </body>
</html>

