

<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>talks</title>
    <meta name="description" content="Karl Broman's talks">
    
    <meta name="author" content="Chiranjib Saha">

    <!-- Enable responsive viewport -->
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le styles -->
    <link href="/assets/themes/twitter/bootstrap/css/bootstrap.2.2.2.min.css" rel="stylesheet">
    <link href="/assets/themes/twitter/css/style.css?body=1" rel="stylesheet" type="text/css" media="all">
    <link href="/assets/themes/twitter/css/kbroman.css" rel="stylesheet" type="text/css" media="all">
    <link href = "maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css",rel = "stylesheet">
    <link href = "maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js",rel = "stylesheet">


    <!-- Le fav and touch icons -->
    <link rel="SHORTCUT ICON" href="/assets/me.ico"/>

    <!-- atom & rss feed -->
    <link href="nil" type="application/atom+xml" rel="alternate" title="Sitewide ATOM Feed">
    <link href="nil" type="application/rss+xml" rel="alternate" title="Sitewide RSS Feed">
  
    <style>
      @import url('https://fonts.googleapis.com/css?family=Lato');
      @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700');
      @import url('https://fonts.googleapis.com/css?family=Montserrat:400,700|PT+Sans');
    </style>
  </head>

  <body>
    <div class="navbar">
      <div class="navbar-inner">
        <div class="container-narrow">
          <a class="brand" href="/">Chiranjib Saha</a>
          <ul class="nav">
              <li><a href="/index.html">About</a></li>
              <li><a href="/pages/news.html">News</a></li>
              <li><a href="/pages/pubs.html">Publications</a></li>
              <!--li><a href="/pages/projects.html">Projects</a></li>
              <li><a href="/pages/talks.html">talks</a></li>
              <li><a href="/pages/software.html">software</a></li>
              <li><a href="/pages/teaching.html">teaching</a></li>
              <li><a href="/pages/tutorials.html">tutorials</a></li>-->
          </ul>
        </div>
      </div>
    </div>

    <div class="container-narrow">

      <div class="content">
        

<div class="page-header">
  <h2>talks </h2>
</div>

<div class="row-fluid">
  <div class="span12">
    <h3 id="2019">2019</h3>

<p>R/qtl2: rewrite of a very old R package
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2_rstudio2019.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_RStudio2019"><img src="icons16/github-icon.png" alt="github" /></a>
<a href="https://resources.rstudio.com/rstudio-conf-2019/r-qtl2-rewrite-of-a-very-old-r-package"><img src="icons16/video-icon.png" alt="video" /></a><br />
<a href="http://www.cvent.com/events/rstudio-conf-austin/event-summary-dd6d75526f3c4554b67c4de32aeffb47.aspx">rstudio::conf</a>,
Austin, Texas (<em>Jan 17</em>)</p>

<h3 id="2018">2018</h3>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/cmp2018.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_CMP2018"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://cmp.wisc.edu">Cellular &amp; Molecular Pathology Graduate Program</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Nov 29</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/steps2rr.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/steps2rr_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.cibm.wisc.edu">CIBM program</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Nov 27</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs2018.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
Presentation in a seminar course for MD-PhD students,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 29</em>)</p>

<p>Cleaning genotype data in diversity outbred mice
<a href="https://kbroman.org/Talk_JAX2018/jax2018.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://kbroman.org/Talk_JAX2018"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_JAX2018"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://churchill-lab.jax.org/website/">Churchill Lab</a> group meeting, <a href="https://www.jax.org">The Jackson Laboratory</a>, Bar Harbor, Maine (<em>Oct 17</em>)</p>

<p>Data management
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/datamgmt.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_DataMgmt"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://ictr.wisc.edu/program/kl2-scholar-program/">KL2 Scholar Program</a>,
<a href="https://www.med.wisc.edu">School of Medicine and Public Health</a>,
<a href="https://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Sep 5</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/steps2rr.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/steps2rr_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://www.biostat.wisc.edu/content/summer-research">Summer Research Program in Biomedical Data Science</a>,
<a href="https://www.biostat.wisc.edu">Department of Biostatistics &amp; Medical Informatics</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jul 11</em>)</p>

<p>Cleaning genotype data in diversity outbred mice
<a href="https://kbroman.org/Talk_CTC2018/broman_ctc2018.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://kbroman.org/Talk_CTC2018"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_CTC2018"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.complextrait.org/ctc2018">Complex Trait Community meeting</a>,
Glasgow, UK (<em>Jun 21</em>)</p>

<p>R/qtl2: QTL mapping in multi-parent populations
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2_purdue2018.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Purdue2018"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.stat.purdue.edu/symp2018">Purdue Symposium on Statistics</a>
(<em>Jun 7</em>)</p>

<p>18 years of R/qtl: maintaining, supporting, and sustaining scientific software<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2_CSU2018.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_CSU2018"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graph:
<a href="https://bit.ly/many_boxplots"><img src="icons16/html-icon.png" alt="many box plots" /></a>
<br />
<a href="http://www.stat.colostate.edu">Department of Statistics</a>,
<a href="https://www.colostate.edu">Colorado State University</a>
(<em>Apr 23</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/steps2rr.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/steps2rr_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://labs.pathology.wisc.edu/oconnor">David O’Connor’s lab</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Mar 19</em>)</p>

<p>18 years of R/qtl: maintaining, supporting, and sustaining scientific software<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2_qBio2018.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_qBio2018"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graph:
<a href="https://bit.ly/many_boxplots"><img src="icons16/html-icon.png" alt="many box plots" /></a>
<br />
<a href="https://events.wid.wisc.edu/events/category/qbio/">qBio seminar
series</a>,
<a href="https://wid.wisc.edu">Wisconsin Institute for Discovery</a>,
University of Wisconsin–Madison (<em>Feb 14</em>)</p>

<p>18 years of R/qtl: developing, maintaining, and supporting scientific software<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2_ucsf2018.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_UCSF2018"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graphs:
<a href="https://bit.ly/lod_and_effect"><img src="icons16/html-icon.png" alt="lod curves" /></a>
<a href="https://bit.ly/many_boxplots"><img src="icons16/html-icon.png" alt="many box plots" /></a>
<br />
<a href="https://biophysics.ucsf.edu/seminars-events/bbc-seminar-series">BBC
seminar</a>,
University of California, San Francisco (<em>Feb 1</em>)</p>

<h3 id="2017">2017</h3>

<p>17 years of R/qtl: maintaining, supporting, and sustaining scientific software<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2_chicago2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Chicago2017"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graphs:
<a href="https://bit.ly/lod_and_effect"><img src="icons16/html-icon.png" alt="lod curves" /></a>
<a href="https://bit.ly/many_boxplots"><img src="icons16/html-icon.png" alt="many box plots" /></a>
<br />
University of Chicago (<em>Nov 17</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
Presentation in a seminar course for MD-PhD students,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 25</em>)</p>

<p>R/qtl2: high-dimensional data &amp; multi-parent populations <br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2_uncc2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_UNCC2017"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graphs:
<a href="https://bit.ly/lod_and_effect"><img src="icons16/html-icon.png" alt="lod curves" /></a>
<a href="https://bit.ly/many_boxplots"><img src="icons16/html-icon.png" alt="many box plots" /></a>
<br />
<a href="http://bioinformatics.uncc.edu/">Department of Bioinformatics and Genomics</a>,
University of North Carolina at Charlotte (<em>Sep 22</em>)</p>

<p>Genetic analysis of high-throughput phenotypes: challenges and opportunities<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/SGN2017/sgn2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_SGN2017"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graphs:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BDS2017-07/lod_and_effect"><img src="icons16/html-icon.png" alt="lod curves" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BDS2017-07/perm_test"><img src="icons16/html-icon.png" alt="permutation test" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/D3/manyboxplots"><img src="icons16/html-icon.png" alt="many box plots" /></a>
<br />
<a href="http://sgn2017.org">Workshop on Systems Genetics of Neurodegenerative Disease</a>,
Frauenchiemsee, Germany (<em>Aug 27</em>)</p>

<p>Genetics of extreme body size evolution in
mice from Gough Island<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BDS2017-07/bds2017-07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BDS2017-07/bds2017-07_withnotes.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<a href="https://github.com/kbroman/Talk_BDS2017-07"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graphs:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BDS2017-07/lod_and_effect"><img src="icons16/html-icon.png" alt="body weight at 5 weeks" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BDS2017-07/perm_test"><img src="icons16/html-icon.png" alt="permutation test" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BDS2017-07/iplot_bodyweight.html"><img src="icons16/html-icon.png" alt="body weight" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BDS2017-07/iplot_deriv_bodyweight.html"><img src="icons16/html-icon.png" alt="growth rate" /></a><br />
<a href="https://www.biostat.wisc.edu/content/summer-research">Summer Research Program in Biomedical Data Science</a>,
<a href="https://www.biostat.wisc.edu">Department of Biostatistics &amp; Medical Informatics</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jul 26</em>)</p>

<p>R/qtl2: high-dimensional data &amp; multi-parent populations
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2_ctc2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Rqtl2_CTC2017"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://complextrait.org/ctc2017">Complex Trait Community meeting</a>,
Memphis, Tennessee (<em>Jun 14</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_BDS2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_BDS2017_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://www.biostat.wisc.edu/content/summer-research">Summer Research Program in Biomedical Data Science</a>,
<a href="https://www.biostat.wisc.edu">Department of Biostatistics &amp; Medical Informatics</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jun 7</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_JAX2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_JAX2017_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
(Remotely to)
<a href="https://www.jax.org/education-and-learning/education-calendar/2017/may/graduate-big-genomic-data-skills-training-for-professors#">Big Genomic Data Skills Training for Graduate Professors</a>,
Jackson Laboratory for Genomic Medicine, Farmington, CT (<em>May 17</em>)</p>

<p>Salvaging a genetics project: Identifying and correcting sample mix-ups in high-dimensional data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/berkeley2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_SampleMixups"><img src="icons16/github-icon.png" alt="github" /></a>
<a href="https://youtu.be/YPd0R-LPI6U"><img src="icons16/video-icon.png" alt="video" /></a><br />
<a href="https://bids.berkeley.edu">Berkeley Institute for Data Science</a>,
<a href="http://www.berkeley.edu">University of California, Berkeley</a> (<em>Apr 7</em>)</p>

<p>Complex trait genetics in mice
<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ProspStudents2017-03/introQTL.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ProspStudents2017-03/introQTL_withnotes.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<a href="https://github.com/kbroman/Talk_ProspStudents"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graphs:
<a href="https://www.biostat.wisc.edu/~kbroman/D3/lod_and_effect"><img src="icons16/html-icon.png" alt="LOD curve" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/D3/cistrans"><img src="icons16/html-icon.png" alt="eQTL plot" /></a><br />
<a href="https://www.biostat.wisc.edu">Biostatistics &amp; Medical Informatics</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Mar 31</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rrworkshop_2017-03.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rrworkshop_2017-03_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/RR_Workshop"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://plantsciencesymposium.umn.edu/workshop/schedule">Data Science and Plant Breeding Simulation Workshop</a>,
University of Minnesota, St. Paul, MN (<em>Mar 23</em>)</p>

<p>Genetics of extreme body size evolution in
mice from Gough Island<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/APStats2017/apstats2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/APStats2017/apstats2017_withnotes.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<a href="https://github.com/kbroman/Talk_APStats"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive graphs:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/APStats2017/lod_and_effect"><img src="icons16/html-icon.png" alt="body weight at 5 weeks" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/APStats2017/perm_test"><img src="icons16/html-icon.png" alt="permutation test" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/APStats2017/iplot_bodyweight.html"><img src="icons16/html-icon.png" alt="body weight" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/APStats2017/iplot_deriv_bodyweight.html"><img src="icons16/html-icon.png" alt="growth rate" /></a><br />
<a href="https://east.madison.k12.wi.us/">Madison East High School</a>, Madison, WI (<em>Mar 9</em>)</p>

<p>Dissection and fine-mapping of <em>trans</em>-eQTL hotspots
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ncsu2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ncsu2017_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_TransHotspots"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://genetics.sciences.ncsu.edu">Genetics Program</a>,
<a href="https://ncsu.edu">North Carolina State University</a>, Raleigh, NC (<em>Feb 6</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs2017.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="http://www.pathology.wisc.edu">Department of Pathology and Laboratory Medicine</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jan 26</em>)</p>

<h3 id="2016">2016</h3>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_ASA2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_ASA2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a>
<a href="http://magazine.amstat.org/videos/education_webinars/ReproducibleResearch.mp4"><img src="icons16/video-icon.png" alt="video" /></a><br />
<a href="https://www.amstat.org/ASA/Education/Web-Based-Lectures.aspx#TRR">Web seminar on Teaching Reproducible Research</a>,
<a href="https://www.amstat.org">American Statistical Association</a> (<em>Nov 16</em>)</p>

<p>R/qtl2: progress &amp; plans
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rqtl2.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Rqtl2"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="http://uthsc.edu">University of Tennessee Health Science Center</a>,
Memphis, Tennessee (<em>Nov 9</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_MDPhD2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs/tree/MDPhD2016"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
Presentation in a seminar course for MD-PhD students,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 26</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_CDHA2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs/tree/CDHA2016"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="http://www.ssc.wisc.edu/cdha/seminars/training.html">Demography Training Seminar</a>,
<a href="http://www.ssc.wisc.edu/cdha/">Center for Demography of Health and Aging</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 5</em>)</p>

<p>Dissection and fine-mapping of <em>trans</em>-eQTL hotspots
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/qrw2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/qrw2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_TransHotspots"><img src="icons16/github-icon.png" alt="github" /></a><br />
Genome and Gene Mapping Satellite, <a href="http://www.queenstownresearchweek.org/qmb-2016/">Queenstown Research Week</a>, Nelson, New Zealand (<em>Sep 1</em>)</p>

<p>Interactive graphics for genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/NZ2016"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_NZ2016"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://www.stat.auckland.ac.nz">Department of Statistics</a>, University of Auckland, New Zealand
(<em>Aug 31</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_JSM2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_JSM2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://www.amstat.org/meetings/jsm/2016/">Joint Statistical Meetings</a>,
Chicago, Illinois (<em>Aug 3</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_TAGC2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_TAGC2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a>
<a href="https://www.youtube.com/watch?v=u-ZgiYjJ4lA"><img src="icons16/video-icon.png" alt="video" /></a><br />
<a href="http://www.genetics2016.org/">The Allied Genetics Conference</a>,
Orlando, Florida (<em>Jul 16</em>)</p>

<p>R/qtl: Just barely sustainable
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/UseR2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/UseR2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_UseR2016"><img src="icons16/github-icon.png" alt="github" /></a>
<a href="https://sec.ch9.ms/ch9/4472/abfcaf43-a51b-4041-a5fb-71eff9514472/D1LYONS3Broman_mid.mp4"><img src="icons16/video-icon.png" alt="video" /></a><br />
<a href="http://user2016.org">UseR 2016 Conference</a>, Palo Alto, California
(<em>Jun 28</em>)</p>

<p>Big data in genetics (<em>10 min session introduction</em>)
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/icqg2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/icqg2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ICQG2016"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.icqg5.org">International Conference on Quantitative Genetics</a>,
Madison, Wisconsin (<em>Jun 16</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_CBB2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_CBB2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://www.biostat.wisc.edu/content/computational-biology-and-biostatistics-summer-research-program-cbb">Computational Biology and Biostatistics Summer Research Program</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jun 7</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_ComBEE2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_ComBEE2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://sites.google.com/a/wisc.edu/combee/">Computational Biology, Ecology &amp; Evolution</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>May 25</em>)</p>

<p>Dissection and fine-mapping of <em>trans</em>-eQTL hotspots
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/cornell2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/cornell2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_TransHotspots"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://bscb.cornell.edu">Department of Biological Statistics and Computational Biology</a>,
<a href="http://www.cornell.edu/">Cornell University</a> (<em>Apr 18</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_UMass2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_UMass2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://gridclub.io">Graduate Researchers interested in Data (GRiD)</a>,
<a href="http://www.umass.edu/">UMass Amherst</a> (<em>Apr 11</em>)</p>

<p>Steps toward reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_ChicagoASA2016.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_ChicagoASA2016_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://community.amstat.org/chicagochapter/home">Chicago Chapter ASA</a>
Conference on <a href="https://s01.123signup.com/servlet/SignUpMember?PG=1531573182300&amp;P=15315731911420461800&amp;Info">Learning Tools to Promote Reproducible Research and Open
Science</a>, Chicago, Illinois (<em>Apr 1</em>)</p>

<p>Interactive graphics for genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/Genentech2016"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_Genentech2016"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.gene.com">Genentech</a>, South San Francisco, California
(<em>Mar 15</em>)</p>

<h3 id="2015">2015</h3>

<p>Interactive graphics for genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ChiASA2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_ChiASA2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://community.amstat.org/chicagochapter">Chicago Chapter of the American Statistical Association</a>, Chicago,
Illinois (<em>Dec 15</em>)</p>

<p>Reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_RDS2015.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_RDS2015_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="http://researchdata.wisc.edu/holz-series/">Holz Series in Research Data Management</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Nov 18</em>)</p>

<p>Reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_Memphis2015.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_Memphis2015_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes/tree/Memphis2015"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="https://www.uthsc.edu/prevmed/biostats.php">Division of Biostatistics</a>,
<a href="https://www.uthsc.edu/prevmed">Department of Preventive Medicine</a>,
<a href="http://uthsc.edu">University of Tennessee Health Science Center</a>,
Memphis, Tennessee (<em>Nov 4</em>)</p>

<p>Interactive graphics for genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BMI2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_BMI2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://www.uthsc.edu/prevmed/biostats.php">Division of Biostatistics</a>,
<a href="https://www.uthsc.edu/prevmed">Department of Preventive Medicine</a>,
<a href="http://uthsc.edu">University of Tennessee Health Science Center</a>,
Memphis, Tennessee (<em>Nov 2</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_MDPhD2014.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs/tree/MDPhD2014"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
Presentation in a seminar course for MD-PhD students,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 28</em>)</p>

<p>Interactive graphics for genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BMI2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_BMI2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://www.biostat.wisc.edu">Department of Biostatistics and Medical Informatics</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 16</em>)</p>

<p>The genetic analysis of high-throughput phenotypes
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/Texas2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_Texas2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://soilcrop.tamu.edu/">Department of Soil and Crop Sciences</a>,
<a href="http://www.tamu.edu/">Texas A&amp;M</a>, College Station, Texas (<em>Sep 1</em>)</p>

<p>Interactive graphics for high-dimensional genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/JSM2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_JSM2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.amstat.org/meetings/JSM/2015">Joint Statistical Meetings</a>,
Seattle, Washington (<em>Aug 10</em>)</p>

<p>Interactive graphics for genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BioC2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_BioC2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://bioconductor.org/help/course-materials/2015/BioC2015/">BioC 2015 conference</a>,
Seattle, Washington (<em>Jul 21</em>)</p>

<p>Reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_SIBS2015.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_SIBS2015_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes/tree/SIBS2015"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="https://www.biostat.wisc.edu/content/summer-institute-training-biostatistics-sibs">Summer Institute for Training in Biostatistics (SIBS)</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jul 9</em>)</p>

<p>Dissecting and fine-mapping <em>trans</em>-eQTL hotspots
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/trans_eqtl_hotspots.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/trans_eqtl_hotspots_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_TransHotspots"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="http://complextrait.org/ctc2015">Complex Trait Community annual meeting</a>,
Portland, Oregon (<em>Jun 11</em>)</p>

<p>Reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes/tree/JAX_CGD"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
Annual retreat of the
<a href="http://cgd.jax.org">Center for Genome Dynamics</a>,
<a href="http://jax.org">Jackson Lab</a>, held in Madison, Wisconsin (<em>May 28</em>)</p>

<p>Reproducible research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/repro_research_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_ReproRes/tree/MoMiEE"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
Molecular Microbial Ecology and Evolution (MoMiEE) group,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>May 11</em>)</p>

<p>Interactive graphics
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/Showcase2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_Showcase2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://quality.wisc.edu/showcase.htm">16th Annual Showcase</a>,
<a href="http://quality.wisc.edu/">Center for Quality Improvement</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Apr 14</em>)</p>

<p>Interactive graphics for high-dimensional genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/WEHI2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_WEHI2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.wehi.edu.au/">Walter and Eliza Hall Institute of Medical Research</a>,
Melbourne, Australia (<em>Apr 1</em>)</p>

<p>Opportunities and challenges in the genetic analysis of
high-throughput phenotypes
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/HTP2015"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_HTP2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
Genome to Phenome Symposium,
<a href="http://www.csiro.au/">CSIRO</a>, Brisbane, Australia (<em>Mar 25</em>)</p>

<p>Interactive graphics for high-dimensional genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/AAAS"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_AAAS2015"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="https://aaas.confex.com/aaas/2015/webprogram/start.html">AAAS Annual Meeting</a>,
San Jose, CA (<em>Feb 13</em>)</p>

<h3 id="2014">2014</h3>

<p>Interactive graphics for teaching
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/iGraphs4Teaching"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_iGraphs4Teaching"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://delta.wisc.edu">Delta Program</a>
<a href="http://delta.wisc.edu/events/roundtable.html">Round Table</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Nov 20</em>)</p>

<p>Interactive graphics for high-dimensional genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/InteractiveGraphs4"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_InteractiveGraphs4"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.hsph.harvard.edu/biostatistics/">Department of Biostatistics</a>,
Harvard School of Public Health, Boston, Massachusetts (<em>Nov 6</em>)</p>

<p>Open access publishing: A researcher’s perspective
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/openaccess2014.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/openaccess2014_withnotes.pdf"><img src="icons16/notes-icon.png" alt="pdf w/notes" /></a>
<a href="https://github.com/kbroman/Talk_OpenAccess"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.library.wisc.edu/news/scholarly-publishing-symposium-2014/">Scholarly Publishing Symposium</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 30</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_MDPhD2014.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs/tree/MDPhD2014"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
Presentation in a seminar course for MD-PhD students,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 29</em>)</p>

<p>How to display data badly
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_cmp2014.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs/tree/cmp2014"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="http://www.pathology.wisc.edu">Department of Pathology and Laboratory Medicine</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Sep 17</em>)</p>

<p>Interactive graphics for
high-dimensional genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/BioVis"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_BioVis"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://biovis.net/year/2014/about">4th Symposium on
Biological Data Visualization</a>, Boston, Massachusetts (<em>Jul 12</em>)</p>

<p>Genetics of extreme body size evolution in
mice from Gough Island<br />
      Slides:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/CTC2014/ctc2014.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/CTC2014/ctc2014_withnotes.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<a href="https://github.com/kbroman/Talk_Gough"><img src="icons16/github-icon.png" alt="github" /></a>
  Interactive LOD curves: <a href="https://www.biostat.wisc.edu/~kbroman/presentations/CTC2014/iplot_bodyweight.html"><img src="icons16/html-icon.png" alt="body weight" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/CTC2014/iplot_deriv_bodyweight.html"><img src="icons16/html-icon.png" alt="growth rate" /></a><br />
<a href="http://www.ctc2014.org">Complex Trait Community 13th annual
meeting</a>, Berlin, Germany (<em>May 21</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ictr2014b.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs/tree/ictr2014"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="http://ictr.wisc.edu">Institute for Clinical and Translational Research</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>May 16</em>)</p>

<p>QTL mapping for phenotypes measured over time
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/FunQTL"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_FunQTL"><img src="icons16/github-icon.png" alt="github" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/FunQTL.tgz"><img src="icons16/tgz-icon.png" alt="tgz" /></a>
<br />
<a href="http://www.danforthcenter.org/">Danforth Plant Science
Center</a>, St. Louis, Missouri (<em>Feb 19</em>)</p>

<p>Creating effective figures and tables
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ictr2014.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs/tree/ictr2014"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
<a href="https://ictr.wisc.edu/SciWritingWorkshop">Manuscript writing
workshop</a>, <a href="http://ictr.wisc.edu">Institute for Clinical and Translational Research</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jan 24</em>)</p>

<h3 id="2013">2013</h3>

<p>How to display data badly (plus a bit on my recent efforts to make interactive graphs) <br />
      Part 1: <a href="https://www.biostat.wisc.edu/~kbroman/presentations/IowaState2013/graphs_combined.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs/tree/iowastate2013"><img src="icons16/github-icon.png" alt="github" /></a>
  Part 2: <a href="https://www.biostat.wisc.edu/~kbroman/presentations/IowaState2013/index.html"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_CGD2013/tree/iowastate2013"><img src="icons16/github-icon.png" alt="github" /></a><br />
(Remotely to) Graphics Working Group, <a href="http://www.stat.iastate.edu">Department of Statistics</a>, <a href="http://www.iastate.edu">Iowa State University</a>, Ames, Iowa (<em>Dec 5</em>)</p>

<p>Open access publishing: A researcher’s perspective
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/openaccess.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/openaccess_withnotes.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<a href="https://github.com/kbroman/Talk_OpenAccess/tree/2013-10"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://researchdata.wisc.edu/open-access-open-data/">Open Access, Open Data @ UW</a> symposium, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 24</em>)</p>

<p>A brief introduction to git &amp; GitHub (with <a href="http://www.stat.wisc.edu/~syounkin/">Samuel G. Younkin</a>)
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/GitPrimer.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/syounkin/GitPrimer"><img src="icons16/github-icon.png" alt="github" /></a><br />
Informal presentation to graduate students, <a href="http://www.stat.wisc.edu">Department of Statistics</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 18</em>)</p>

<p>How to display data badly
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_combined.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_Graphs"><img src="icons16/github-icon.png" alt="github" /></a><br />
Lecture to MD/PhD students, <a href="http://www.med.wisc.edu/">School of Medicine and Public Health</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 16</em>)</p>

<p>Interactive graphics for high-dimensional genetic data  <a href="https://www.biostat.wisc.edu/~kbroman/presentations/CGD2013/presentation.html"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_CGD2013"><img src="icons16/github-icon.png" alt="github" /></a><br />
Jackson Laboratory, <a href="http://cgd.jax.org">Center for Genome Dynamics</a>, Bar Harbor, Maine (<em>Jun 25</em>)</p>

<p>MAGIC design and other topics
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/magic_web.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://github.com/kbroman/Talk_MAGIC"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://mus.well.ox.ac.uk/19genomes/MAGIC-WORKSHOP/">Workshop on MAGIC-type populations</a> in Cambridge, UK (<em>Jun 12</em>)</p>

<p>Interactive graphics for high-dimensional genetic data  <a href="https://www.biostat.wisc.edu/~kbroman/presentations/CTC2013"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_CTC2013"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://rqtl.org/ctc2013">Complex Trait Community 12th annual meeting</a>, Madison, Wisconsin (<em>May 31</em>)</p>

<p>Why aren’t all of our graphs interactive?  <a href="https://www.biostat.wisc.edu/~kbroman/presentations/InteractiveGraphs3"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_InteractiveGraphs3"><img src="icons16/github-icon.png" alt="github" /></a>
<a href="InteractiveGraphs3.tgz"><img src="icons16/tgz-icon.png" alt="tgz" /></a><br />
Biostatistics training program, <a href="https://www.biostat.wisc.edu">Department of Biostatistics and Medical Informatics</a>, <a href="http://www.med.wisc.edu">School of Medicine and Public Health</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Apr 8</em>)</p>

<p>QTL mapping for phenotypes measured over time
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/FunQTL"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_FunQTL"><img src="icons16/github-icon.png" alt="github" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/FunQTL.tgz"><img src="icons16/tgz-icon.png" alt="tgz" /></a>
<br />
Plant Breeding Symposium, <a href="http://www.agronomy.ksu.edu">Department of Agronomy</a>, <a href="http://www.ksu.edu">Kansas State University</a>, Manhattan, KS (<em>Apr 2</em>)</p>

<p>Interactive graphics for high-dimensional genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/InteractiveGraphs2"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_InteractiveGraphs2"><img src="icons16/github-icon.png" alt="github" /></a>
<br />
International Biometric Society/ENAR meeting, Orlando, FL (<em>Mar 13</em>)</p>

<p>Identifying and correcting sample mix-ups in high-dimensional genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/anisci2013.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://http://www.ansci.wisc.edu">Department of Animal Sciences</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Feb 12</em>)</p>

<p>Identifying and correcting sample mix-ups in high-dimensional genetic data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/bmi2013.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="https://www.biostat.wisc.edu">Department of Biostatistics and Medical Informatics</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Feb 8</em>)</p>

<h3 id="2012">2012</h3>

<p>How to display data badly
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_uwpath2012.pptx"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<br />
<a href="http://www.pathology.wisc.edu">Department of Pathology and Laboratory Medicine</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 17</em>)</p>

<p>Why aren’t all of our graphs interactive?  <a href="https://www.biostat.wisc.edu/~kbroman/presentations/InteractiveGraphs"><img src="icons16/html-icon.png" alt="html" /></a>
<a href="https://github.com/kbroman/Talk_InteractiveGraphs1"><img src="icons16/github-icon.png" alt="github" /></a><br />
<a href="http://www.biostat.jhsph.edu">Department of Biostatistics</a>, <a href="http://www.jhsph.edu">Johns Hopkins Bloomberg School of Public Health</a>, Baltimore, MD (<em>Oct 5</em>)</p>

<p>Identifying and correcting sample mix-ups in high-dimensional data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/evolsysbiol.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a><br />
<a href="http://evolutionarysystemsbiology.org/meeting/2012-Madison/index.html">EvoSysBio meeting</a>, Wisconsin Institutes for Discovery, University of Wisconsin–Madison (<em>Jul 30</em>)</p>

<p>Identifying and correcting sample mix-ups in high-dimensional data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/germany2012.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a><br />
<a href="http://www.euratrans.eu">EURATRANS</a> meeting, Tutzing, Germany (<em>Jun 4</em>)</p>

<p>Identifying and correcting sample mix-ups in high-dimensional data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/enar2012.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a><br />
International Biometric Society/ENAR meeting, Washington, DC (<em>Apr 2</em>)</p>

<p>Programming style
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/programming_style.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Biostatistics training program, <a href="https://www.biostat.wisc.edu">Department of Biostatistics and Medical Informatics</a>, <a href="http://www.med.wisc.edu">School of Medicine and Public Health</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Feb 29</em>)</p>

<h3 id="2011">2011</h3>

<p>Salvaging a genetics project: Identifying and correcting sample mix-ups in high-dimensional data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/washbiostat2011.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.biostat.washington.edu">Department of Biostatistics</a>, <a href="http://www.washington.edu">University of Washington</a>, Seattle (<em>Nov 17</em>)</p>

<p>Salvaging an eQTL project: Identifying and correcting sample mix-ups
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/gmu2011.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://statistics.gmu.edu">Department of Statistics</a>, <a href="http://www.gmu.edu">George Mason University</a>, Fairvax, VA (<em>Sep 23</em>)</p>

<p>Salvaging an eQTL project: Identifying and correcting sample mix-ups
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/groningen2011.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.rug.nl/fmns-research/bioinformatics/index">Groningen Bioinformatics Centre</a>, <a href="http://www.rug.nl">University of Groningen</a>, Groningen, The Netherlands (<em>Sep 8</em>)</p>

<p>Identifying sample mix-ups in eQTL data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/mousegenet2011.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.mousegenetics2011.org">Mouse Genetics 2011</a>, Washington, DC (<em>Jun 23</em>)</p>

<p>Mapping quantitative trait loci in mice
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/multiqtl_ireland11.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.qub.ac.uk/sites/QuantBio/">Quantitative Biology and Bioinformatics in Modern Medicine</a>, Dublin, Ireland (<em>Feb 7</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/multiqtl_columbia11.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.mailman.columbia.edu/academic-departments/biostatistics">Department of Biostatistics</a>, <a href="http://www.mailman.columbia.edu">Mailman School of Public Health</a>, <a href="http://www.columbia.edu">Columbia University</a>, New York, NY (<em>Jan 27</em>)</p>

<h3 id="2010">2010</h3>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/multiqtl_poland10.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://pwr.edu.pl/en_main.xml">Wrocław University of Technology</a>, Wrocław, Poland (<em>May 21</em>)</p>

<p>Genetic maps: Past, present and future
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/geneticmaps_poland10.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.up.wroc.pl/en/">Wrocław University of Environmental and Life Sciences</a>, Wrocław, Poland (<em>May 20</em>)</p>

<p>The genetic dissection of complex traits in model organisms
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/qtlmas10.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://jay.up.poznan.pl/qtlmas2010/">14th QTL-MAS Workshop</a>, Poznań, Poland (<em>May 17</em>)</p>

<p>Mapping QTL to a phylogenetic tree
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ctc2010.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.ctc2010.org">9th annual meeting</a> of the <a href="http://www.complextrait.org">Complex Trait Consortium</a>, Chicago, IL (<em>May 10</em>)</p>

<p>How to give a scientific presentation
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/giving_talks.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Biostatistics training program, <a href="https://www.biostat.wisc.edu">Department of Biostatistics and Medical Informatics</a>, <a href="http://www.med.wisc.edu">School of Medicine and Public Health</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Apr 5</em>)</p>

<p>Mapping QTL to a phylogenetic tree
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/michstate10.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.pa.msu.edu/seminars/edge/">Edge Seminar series</a>, <a href="http://biomodel.msu.edu/">Quantitative Biology and Modeling Initiative Program</a>, <a href="http://www.msu.edu">Michigan State University</a>, East Lansing, MI (<em>Apr 2</em>)</p>

<p>Mapping QTL to a phylogenetic tree
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/vanderbilt10.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Integrative Genomics Seminar Series, <a href="http://www.vanderbilt.edu">Vanderbilt University</a>, Nashville, TN (<em>Mar 9</em>)</p>

<p>Mapping QTL to a phylogenetic tree
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/uwevolution10.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.evolution.wisc.edu/seminars.php">Evolution seminar series</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Mar 4</em>)</p>

<p>Mapping QTL to a phylogenetic tree
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/unc10.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://gmb.unc.edu">Curriculum in Genetics and
Molecular Biology</a>, <a href="http://www.unc.edu">University of
North Carolina at Chapel Hill</a> (<em>Feb 12</em>)</p>

<h3 id="2009">2009</h3>

<p>Mapping QTL to a phylogenetic tree
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/uwgenetics09.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.genetics.ucla.edu">Department of
Human Genetics</a>, <a href="http://www.ucla.edu">University of
California, Los Angeles</a> (<em>Nov 23</em>)</p>

<p>Mapping QTL to a phylogenetic tree
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ucla09.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.genetics.wisc.edu">Laboratory of
Genetics</a>, <a href="http://www.wisc.edu">University of
Wisconsin–Madison</a> (<em>Nov 18</em>)</p>

<p>An applied statistician does probability: It’s not pretty
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/uwstat09.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.stat.wisc.edu/">Department of Statistics</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 7</em>)</p>

<p>How to display data badly
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_uwpath09.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<br />
<a href="http://www.pathology.wisc.edu">Department of Pathology and Laboratory Medicine</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Sep 30</em>)</p>

<p>Genetic maps: Past, present and future
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/chair09.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="https://www.biostat.wisc.edu">Department of Biostatistics and Medical Informatics</a>,
<a href="http://www.med.wisc.edu/">School of Medicine and Public Health</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Sep 17</em>)</p>

<p>Genotype probabilities in the pre-CC
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/jax_cgd_09.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br /><a href="http://cgd.jax.org/">Center for Genome Dynamics</a>, <a href="http://www.jax.org">The Jackson Laboratory</a>, Bar Harbor, Maine (via phone) (<em>Sep 15</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/oxford09.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.well.ox.ac.uk/">Wellcome Trust Centre for Human Genetics</a>, Oxford, United Kingdom (<em>Jan 23</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/groningen09.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.rug.nl/fmns-research/bioinformatics/index">Groningen Bioinformatics Centre</a>, <a href="http://www.rug.nl">University of Groningen</a>, Groningen, The Netherlands (<em>Jan 20</em>)</p>

<h3 id="2008">2008</h3>

<p>Crossover interference and the sex difference in recombination
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/uwstat08.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.stat.wisc.edu">Department of Statistics</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Sep 10</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/gstpretreat08.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.gstp.wisc.edu">Genomic Sciences Training Program</a> <a href="http://www.gstp.wisc.edu/events/retreat.htm">annual retreat</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jun 9</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/banff08.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.birs.ca/birspages.php?task=displayevent&amp;event_id=08w5062">Emerging Statistical Challenges In Genome and Translational Research</a>, Banff, Canada (<em>Jun 2</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/cincinnati08.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.cincinnatichildrens.org/svc/alpha/h/genetics/default.htm">Division of Human Genetics</a>, <a href="http://www.cincinnatichildrens.org">Cincinnati Children’s Hospital</a> (<em>Apr 25</em>)</p>

<p>Genetic maps: Past, present and future
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/cibm08.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.cibm.wisc.edu">Computation and Informatics in Biology and Medicine (CIBM)</a> <a href="http://www.cibm.wisc.edu/seminars.html">seminar series</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Apr 22</em>)</p>

<p>Genetic maps: Past, present and future
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/geneticmaps08.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.evolution.wisc.edu/seminars.php">Evolution seminar series</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Feb 21</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/animalbreeding08.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/animalbreeding08_bw-2x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
Graduate Seminar, <a href="http://www.anisci.wisc.edu">Department of Animal Sciences</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Feb 12</em>)</p>

<p>How to display data badly
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_uwpath08.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_uwpath08_handout.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.pathology.wisc.edu">Department of Pathology and Laboratory Medicine</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Jan 30</em>)</p>

<h3 id="2007">2007</h3>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/uchicago07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/uchicago07_bw-2x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br /><a href="http://www.genes.uchicago.edu">Department of Human Genetics</a>, <a href="http://www.uchicago.edu">University of Chicago</a> (<em>Dec 5</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/hopkins07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/hopkins07_bw-2x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br /><a href="http://www.biostat.jhsph.edu">Department of Biostatistics</a>, <a href="http://www.jhsph.edu">Johns Hopkins Bloomberg School of Public Health</a>, Baltimore, MD (<em>Nov 28</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/stolaf07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/stolaf07_bw-2x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br /><a href="http://www.stolaf.edu/depts/statistics/">Statistics Program</a>, <a href="http://www.stolaf.edu">St. Olaf College</a>, Northfield, MN (<em>Nov 6</em>)</p>

<p>How to display data badly
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_stolaf07.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<br /><a href="http://www.stolaf.edu/academics/cir/">Center for Interdisciplinary Research</a>, <a href="http://www.stolaf.edu">St. Olaf College</a>, Northfield, MN (<em>Nov 5</em>)</p>

<p>Crossover interference and the sex difference in recombination
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/geneticsRetreat07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/geneticsRetreat07_bw-2x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
Annual Retreat, <a href="http://www.genetics.wisc.edu">Laboratory of Genetics</a>,
<a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Oct 5</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/niu07b.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/niu07b_bw-2x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.math.niu.edu/StatDiv">Division of Statistics</a>,
<a href="http://www.math.niu.edu">Department of Mathematical Sciences</a>,
<a href="http://www.niu.edu">Northern Illinois University</a>, DeKalb, IL (<em>Sep 11</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/niu07a.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.math.niu.edu/StatDiv">Division of Statistics</a>,
<a href="http://www.math.niu.edu">Department of Mathematical Sciences</a>,
<a href="http://www.niu.edu">Northern Illinois University</a>, DeKalb, IL (<em>Sep 10</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/wisc07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/wisc07_bw-2x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="https://www.biostat.wisc.edu">Department of
Biostatistics and Medical Informatics</a>, <a href="http://www.wisc.edu">University of
Wisconsin–Madison</a> (<em>Mar 22</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/la07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.genetics.ucla.edu">Department of
Human Genetics</a>, <a href="http://www.ucla.edu">University of
California, Los Angeles</a> (<em>Mar 5</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/berkeley07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.stat.berkeley.edu">Department of
Statistics</a>, <a href="http://www.berkeley.edu">University of
California, Berkeley</a> (<em>Feb 27</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/berkeley07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.jax.org">The Jackson Laboratory</a>, Bar
Harbor, Maine (<em>Feb 22</em>)</p>

<p>Mapping multiple QTL in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/mich07.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/mich07_bw-2x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.sph.umich.edu/biostat">Department of
Biostatistics</a>, <a href="http://www.umich.edu">University of
Michigan</a>, Ann Arbor (<em>Feb 15</em>)</p>

<p>QTL mapping in humans
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/humanqtl.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/humanqtl_1x2.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
Systems Medicine Workshop, NHLBI, Bethesda, MD (<em>Jan 31</em>)</p>

<h3 id="2006">2006</h3>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/hopkins_igm06.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.hopkinsmedicine.org/geneticmedicine/">McKusick-Nathans Institute of Genetic Medicine</a>, <a href="http://www.hopkinsmedicine.org">School of Medicine</a>, Johns Hopkins University, Baltimore, MD (<em>Nov 9</em>)</p>

<p>The X chromosome in QTL mapping: What a pain in the @$$!
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/xchr.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.biostat.jhsph.edu/GenomeCAFE/ExpressionistSchedule.html">Hopkins Expressionists</a> (<em>Nov 1</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/michigan06.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/michigan06_handout.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.sph.umich.edu/biostat">Department of
Biostatistics</a>, <a href="http://www.umich.edu">University of
Michigan</a>, Ann Arbor (<em>Oct 19</em>)</p>

<p>Poor performance of bootstrap confidence
intervals for QTL location
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/qtlboot.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.complextrait.org">Complex Trait
Consortium</a> <a href="http://www.complextrait.org/meetings/ctc06.php">5th annual meeting</a>, Chapel Hill, North Carolina (<em>May 7</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/wisc06.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/wisc06_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.genetics.wisc.edu">Genetics</a>, <a href="http://www.wisc.edu">University of Wisconsin–Madison</a> (<em>Apr 5</em>)</p>

<p>Principles of experimental design and sample
size determination for animal-based research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/acuc.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/acuc_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.jhu.edu/animalcare">Johns Hopkins University Animal Care
and Use Committee</a> seminar (<em>Feb 23</em>)</p>

<p>Model selection for multiple interacting QTL
in experimental crosses
<br />
<a href="http://www.biostat.jhsph.edu/GenomeCAFE/ExpressionistSchedule.html">Hopkins Expressionists</a> (<em>Feb 15</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/seattle06.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/seattle06_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.biostat.washington.edu">Department of
Biostatistics</a>, <a href="http://www.washington.edu">University of Washington</a>, Seattle (<em>Jan 12</em>)</p>

<h3 id="2005">2005</h3>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/oz05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/oz05_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://bioinf.wehi.edu.au/folders/melanie/genemappers/index.htm">Fifth Australasian human gene mapping conference</a>, Mt. Buller, Australia (<em>Nov 23</em>)</p>

<p>What is regression?
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/regression.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/regression_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
Part of the “<a href="http://www.biostat.jhsph.edu/newsEvent/event/lectureSeries/lecture2005-06.php">Everything you ever wanted to know about
Biostatistics but were afraid to ask</a>” lecture series<br />
<a href="http://www.biostat.jhsph.edu">Department of
Biostatistics</a>, <a href="http://www.jhsph.edu">Johns Hopkins
Bloomberg School of Public Health</a>, Baltimore, MD (<em>Oct 21</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ucdavis05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ucdavis05_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.stat.ucdavis.edu">Department of
Statistics</a>, <a href="http://www.ucdavis.edu">University of
California, Davis</a> (<em>Sep 29</em>)</p>

<p>Super computing for “classical” genetics
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/genomecafe.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/genomecafe_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.biostat.jhsph.edu/GenomeCAFE">Genome Cafe</a>
Grand Opening, <a href="http://www.biostat.jhsph.edu">Department of
Biostatistics</a>, <a href="http://www.jhsph.edu">Bloomberg School
of Public Health</a>, <a href="http://www.jhu.edu">Johns Hopkins
University</a>, Baltimore, MD (<em>Sep 14</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/upenn05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/upenn05_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.med.upenn.edu/genetics">Department of Genetics</a>, <a href="http://www.med.upenn.edu/">School of
Medicine</a>, <a href="http://www.upenn.edu">University of
Pennsylvania</a>, Philadelphia, PA (<em>Sep 12</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/csps05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/csps05_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.amstat.org/meetings/jsm/2005/index.cfm">Joint
Statistical Meetings</a>, Minneapolis, MN (<em>Aug 11</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/csps05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/csps05_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://math.bnu.edu.cn/statprob/CSPS-IMS2005/index.html">CSPS/IMS Joint meeting</a>, Beijing, China (<em>Jul 10</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/umbc05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/umbc05_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.math.umbc.edu">Department of Mathematics and Statistics</a>,
<a href="http://www.umbc.edu">University of Maryland, Baltimore County</a>, Baltimore, MD (<em>Apr 8</em>)</p>

<p>Understanding human disease via randomized mice
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/epi05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/epi05_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.jhsph.edu/Dept/EPI/index.html">Department of
Epidemiology</a>, <a href="http://www.jhsph.edu">Johns Hopkins
Bloomberg School of Public Health</a>, Baltimore, MD (<em>Mar 4</em>)</p>

<p>Principles of experimental design and sample
size determination for animal-based research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/acuc.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/acuc_ho.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.jhu.edu/animalcare">Johns Hopkins University Animal Care
and Use Committee</a> seminar (<em>Feb 24</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/prosp_students05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/prosp_students05.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
(for prospective graduate students)<br />
<a href="http://www.biostat.jhsph.edu">Department of
Biostatistics</a>, <a href="http://www.jhsph.edu">Johns Hopkins
Bloomberg School of Public Health</a>, Baltimore, MD (<em>Feb 11</em>)</p>

<p>The genetic dissection of complex traits
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/martina_defense.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/martina_defense.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
(Introduction for the Martina Johannesson’s dissertation defense)<br />
<a href="http://www.lu.se/o.o.i.s/1705?task=view_english_organization_information013100009&amp;firstname=&amp;lastname=holmdahl">Section for Medical Inflammation Research</a>, <a href="http://www.lu.se/o.o.i.s/1705?task=view_english_organization_information013100007&amp;username=">Department of Cell and Molecular Biology</a>, <a href="http://www.lu.se/o.o.i.s/450">Lund University</a>, Lund, Sweden (<em>Jan 28</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/lund05.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/lund05.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a>
<br />
<a href="http://www.lu.se/o.o.i.s/1705?task=view_english_organization_information013100009&amp;firstname=&amp;lastname=holmdahl">Section for Medical Inflammation Research</a>, <a href="http://www.lu.se/o.o.i.s/1705?task=view_english_organization_information013100007&amp;username=">Department of Cell and Molecular Biology</a>, <a href="http://www.lu.se/o.o.i.s/450">Lund University</a>, Lund, Sweden (<em>Jan 27</em>)</p>

<h3 id="2004">2004</h3>

<p>Gene mapping in model organisms
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/taiwan04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/taiwan04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.stat.sinica.edu.tw/2004tssg/">Taipei Symposium
on Statistical Genomics</a>, Academia Sinica, Taipei, Taiwan (<em>Dec 16</em>)</p>

<p>The genetic dissection of complex
traits
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/tigr04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/tigr04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.tigr.org/conf/cg/">7th Annual Conference on
Computational Genomics</a>, Reston, Virginia (<em>Oct 21</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/yale04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/yale04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://publichealth.yale.edu/bis/index.html">Department of
Biostatistics</a>, <a href="http://publichealth.yale.edu/">Yale
School of Public Health</a>, New Haven, Connecticut (<em>Oct 19</em>)</p>

<p>Identifying essential genes in <em>M. tuberculosis</em>
by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/mfd04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/mfd04_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://research.marshfieldclinic.org/">Marshfield Clinic
Research Foundation</a>, Marshfield, Wisconsin (<em>Oct 5</em>)</p>

<p><a name="wow"></a>How to display data badly
[<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_photos">Photos from the event</a>]
<br />
           
Version presented:
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_final.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a><br />
           
Director’s cut: <a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_win.ppt"><img src="icons16/ppt-icon.png" alt="windows ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs_mac.ppt"><img src="icons16/ppt-icon.png" alt="Mac ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/graphs.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a><br />
Part of the “<a href="http://www.biostat.jhsph.edu/newsEvent/event/lecture.php">Everything you ever wanted to know about
Biostatistics but were afraid to ask</a>” lecture series<br />
<a href="http://www.biostat.jhsph.edu">Department of
Biostatistics</a>, <a href="http://www.jhsph.edu">Johns Hopkins
Bloomberg School of Public Health</a>, Baltimore, MD (<em>Sep 24</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/toronto04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/toronto04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.sickkids.on.ca/Research/section.asp?s=Research+Programs&amp;sID=1118&amp;ss=Genetics+and+Genomic+Biology&amp;ssID=2504">Genetics and Genomic Biology</a>,
<a href="http://www.sickkids.on.ca">Hospital for Sick Children</a>, Toronto, Canada (<em>Aug 11</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/jsm04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/jsm04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.amstat.org/meetings/jsm/2004/index.cfm">Joint
Statistical Meetings</a>, Toronto, Canada (<em>Aug 8</em>)</p>

<p>The genomes of recombinant inbred lines
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ctc04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ctc04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.complextrait.org">Complex Trait Consortium</a>, <a href="http://www.jax.org/courses/events/coursedetails.do?id=30">Third Annual Meeting</a>, <a href="http://www.jax.org">The Jackson Laboratory</a>, Bar Harbor, Maine (<em>Jul 8</em>)</p>

<p>Statistics in epigenomics
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/epigenomics.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/epigenomics.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www-conference.slu.se/nobel/">Nobel Symposium on
Epigenetic Reprogramming in Development
and Disease</a>, Stockholm, Sweden (<em>Jun 19</em>)</p>

<p>The breakpoint process on recombinant inbred
chromosomes
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/vanderbilt04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/vanderbilt04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Genetic Interest Group, <a href="http://phg.mc.vanderbilt.edu/">Center for Human Genetics
Research</a>, <a href="http://www.vanderbilt.edu">Vanderbilt
University</a>, Nashville, TN (<em>May 27</em>)</p>

<p>The breakpoint process on recombinant inbred chromosomes
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/retreat04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/retreat04_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Annual Retreat, <a href="http://www.biostat.jhsph.edu">Department of
Biostatistics</a>, <a href="http://www.jhsph.edu">Johns Hopkins Bloomberg School of
Public Health</a> (<em>May 1</em>)</p>

<p>The breakpoint process on recombinant inbred chromosomes
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ribreaks.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ribreaks_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://astor.som.jhmi.edu/hex">Hopkins Expressionists</a> (<em>Apr 21</em>)</p>

<p>Identifying essential genes in <em>M. tuberculosis</em>
by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/sk04_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://phhp.buffalo.edu/biostat">Dept of Biostatistics,
University at Buffalo</a> in Buffalo, New York (<em>Apr 8</em>)</p>

<p>Introduction to QTL mapping: lecture and
computer lab
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/Louisville/rqtllab.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/Louisville/rqtllab.R"><img src="icons16/R-icon.png" alt="R code" /></a><br />
<a href="https://www.sphis.louisville.edu/bb_home.cfm">Dept of
Bioinformatics and Biostatistics</a>, <a href="http://www.louisville.edu">University of Louisville</a> in
Louisville, Kentucky (<em>Apr 2</em>)</p>

<p>Statistical issues in QTL mapping in mice
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rostock04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rostock04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.eurome.de/meet.html#meet4">Workshop on the
Analysis of Complex Genetic Traits</a>, <a href="http://www-iimmun.med.uni-rostock.de/immgen.html">Immunogenetics</a>, <a href="http://www.uni-rostock.de">Universität Rostock</a>, Germany (<em>Mar 17</em>)</p>

<p>Gene mapping in mice
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/msri04.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/msri04.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="http://www.msri.org/publications/ln/msri/2004/genetics/broman/1/index.html"><img src="icons16/video-icon.png" alt="video" /></a>
<br />
<a href="http://www.msri.org/calendar/workshops/WorkshopInfo/261/show_workshop">Workshop on the Genetics of Complex Disease</a>, <a href="http://www.msri.org">Mathematical Sciences
Research Institute</a>, Berkeley, California (<em>Feb 12</em>)</p>

<p>Identifying essential genes in <em>M. tuberculosis</em>
by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/sk04_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.mskcc.org/mskcc/html/3159.cfm">Department of
Epidemiology and Biostatistics</a>, <a href="http://www.mskcc.org/mskcc/html/44.cfm">Memorial
Sloan-Kettering Cancer Center</a>, New York (<em>Jan 14</em>)</p>

<h3 id="2003">2003</h3>
<p>Identifying essential genes in <em>M. tuberculosis</em>
by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ucla03_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Departments of <a href="http://www.stat.wisc.edu/">Statistics</a>
and <a href="https://www.biostat.wisc.edu/">Biostatistics &amp; Medical Informatics</a>, University of Wisconsin–Madison (<em>Nov 12</em>)</p>

<p>Identifying essential genes in <em>M. tuberculosis</em>
by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ucla03_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.stat.ucla.edu/">Department of Statistics</a>,
University of California, Los Angeles (<em>Nov 4</em>)</p>

<p>Identifying essential genes in <em>M. tuberculosis</em>
by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/unc03_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.sph.unc.edu/bios">Department of Biostatistics</a>, University of North
Carolina at Chapel Hill (<em>Oct 23</em>)</p>

<p>Identifying essential genes in <em>M. tuberculosis</em>
by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/haverford_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Haverford-Bryn Mayr Mathematics Colloquium, <a href="http://www.haverford.edu/math/">Haverford College</a>, Haverford, Pennsylvania (<em>Oct 20</em>)</p>

<p>Identifying essential genes in <em>M. tuberculosis</em>
by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/jsm03_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Joint Statistical Meetings, San Francisco (<em>Aug 5</em>)</p>

<p>Principles of experimental design and sample
size determination for animal-based research
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/acuc.ppt"><img src="icons16/ppt-icon.png" alt="ppt" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/acuc_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.jhu.edu/animalcare">Johns Hopkins University Animal Care
and Use Committee</a> seminar (<em>Jun 26</em>)</p>

<p>Identifying essential genes in
<em>M. tuberculosis</em> by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ucsf03_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://astor.som.jhmi.edu/hex">Hopkins Expressionists</a> (<em>Jun 25</em>)</p>

<p>Identifying essential genes in
<em>M. tuberculosis</em> by random transposon mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ucsf03_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.biostat.ucsf.edu/">Department of Biostatistics</a>, University of California, San Francisco (<em>Mar 13</em>)</p>

<p>Meiosis, recombination, and
interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/meiosis_nz.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/meiosis_nz_handout.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a><br />
QTL mapping in mice
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/qtl_nz.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/qtl_nz_handout.pdf"><img src="icons16/notes-icon.png" alt="handout" /></a><br />
<a href="http://www.mcs.vuw.ac.nz/~mathmeet/newplymouth/index.shtml">Mathematics
Workshop</a> in New Plymouth, New Zealand (<em>Jan 10-11</em>)</p>

<h3 id="2002">2002</h3>

<p>Identification of the essential genes in the
<em>M. tuberculosis</em> genome by random transposon
mutagenesis
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/mtb_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.biostat.jhsph.edu">Department of
Biostatistics</a>, <a href="http://www.jhsph.edu">Johns Hopkins Bloomberg School of
Public Health</a> (<em>Dec 13</em>)</p>

<p>Human crossover interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/yale.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<!--[![video](icons16/video-icon.png)](http://images.main.uab.edu/isoph/PresenterOne/HumanCrossoverInterference/rnhigh.ram)--> <br />
<a href="http://www.soph.uab.edu/statgenetics/Default.htm">Section on Statistical Genetics</a>, University of Alabama,
Birmingham (<em>Oct 9</em>)</p>

<p>Model selection for QTL mapping
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rss_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a><br />
Joint Statistical Meetings, New York, NY (<em>Aug 14</em>)</p>

<p>Model selection for QTL mapping
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rss_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Royal Statistical Society, London, England (<em>May 22</em>)</p>

<p>R/qtl: an extensible QTL mapping environment
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/ctc.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Complex Trait Consortium meeting, Memphis, Tennessee (<em>May 16</em>)</p>

<p>Human crossover interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/yale.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.stat.berkeley.edu">Department of Statistics</a>, University of California, Berkeley (<em>Apr 11</em>)</p>

<p>A model selection approach for the identification of quantitative
trait loci in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/hopkins02_ho.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.biostat.jhsph.edu">Department of Biostatistics</a>,
Johns Hopkins University, Baltimore, Maryland (<em>Apr 3</em>)</p>

<p>Model selection in QTL mapping
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/rp_handout.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.roswellpark.org/department_725.html">Department of Molecular and
Cellular Biology</a>, Roswell Park Cancer Institute, Buffalo, New York (<em>Mar 28</em>)</p>

<p>A model selection approach for the identification of quantitative
trait loci in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/enar02_handout.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
International Biometric Society/ENAR meeting, Arlington, Virginia (<em>Mar 20</em>)</p>

<p>Human crossover interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/yale.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.american.edu/cas/department_mathstat.shtml">Department of Mathematics and
Statistics</a>, American University, Washington, DC (<em>Feb 5</em>)</p>

<h3 id="2001">2001</h3>

<p>Human crossover interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/yale.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.stat.yale.edu/">Department of Statistics</a>, Yale
University, New Haven, Connecticut (<em>Dec 3</em>)</p>

<p>Identifying clusters of related individuals with data on dominant genetic markers
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/stlouis01.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Classification Society of North America meeting, St. Louis, Missouri (<em>Jun 15</em>)</p>

<p>QTL mapping in mice: Review of single QTL methods
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/madison01.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
NCI Satellite Workshop on Modifier Genes, Madison, Wisconsin (<em>Jun 12</em>)</p>

<p>Human meiotic interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/uwm2001.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.math.uwm.edu">Department of Mathemetical Sciences</a>,
University of Wisconsin-Milwaukee (<em>May 11</em>)</p>

<p>Crossover interference in the mouse
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/mouseinterfer.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Hopkins Statistical Genetics Working Group (<em>Apr 2</em>)</p>

<h3 id="2000">2000</h3>

<p>Human crossover interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/jax.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://linus.nci.nih.gov/~brb/">Biometric Research Branch</a>, National Cancer Institute, Bethesda, Maryland (<em>Oct 20</em>)</p>

<p>Identifying QTLs in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/hsgwg.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Hopkins Statistical Genetics Working Group (<em>Sep 25</em>)</p>

<p>Human crossover interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/jax.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.jax.org">The Jackson Laboratory</a>, Bar Harbor, Maine (<em>Sep 15</em>)</p>

<p>Estimation of allele frequencies with sibpair data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/enar2000.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
International Biometric Society/ENAR meeting, Chicago, IL (<em>Mar 20</em>)</p>

<h3 id="1999">1999</h3>

<p>Topics in statistical genetics
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/topics.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.biostat.jhsph.edu">Department of Biostatistics</a>, Johns Hopkins University, Baltimore, Maryland (<em>Sep 22</em>)</p>

<p>A characterization of human crossover interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/enar1999.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
International Biometric Society/ENAR meeting, Atlanta, GA (<em>Mar 30</em>)</p>

<p>Human meiotic interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/interference.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://depts.washington.edu/biostat/">Department of Biostatistics</a>, <a href="http://www.washington.edu">University of Washington</a>, Seattle (<em>Feb 18</em>)</p>

<p>Identifying QTLs in experimental crosses
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/qtls.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
Gemini Research, Cambridge, England (<em>Feb 11</em>)</p>

<p>Human meiotic interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/interference.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.biostat.jhsph.edu">Department of Biostatistics</a>, Johns Hopkins
University, Baltimore, Maryland (<em>Feb 3</em>)</p>

<p>Human meiotic interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/interference.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a><br />
<a href="http://www.stat.berkeley.edu">Department of Statistics</a>, University
of California, Berkeley (<em>Jan 28</em>)</p>

<p>Human meiotic interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/interference.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://mcb.berkeley.edu/faculty/GEN/rinej.html">Jasper Rine</a>’s laboratory,
Molecular and Cell Biology,
University of California, Berkeley (<em>Jan 27</em>)</p>

<p>Human meiotic interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/interference.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://research.marshfieldclinic.org/genetics/">Center for Medical Genetics</a>,
Marshfield Medical Research Foundation, Marshfield, Wisconsin (<em>Jan 14</em>)</p>

<h3 id="1998">1998</h3>

<p>Human meiotic interference
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/interference.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.wehi.edu.au/">Walter and Eliza Hall Institute for Medical Research</a>,
Melbourne, Australia (<em>Nov 24</em>)</p>

<p>Cleaning genotype data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/cleaning.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.qimr.edu.au/">Queensland Institute for Medical Research</a>,
Brisbane, Australia (<em>Nov 19</em>)</p>

<p>Cleaning genotype data
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/cleaning.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://www.wehi.edu.au/">Walter and Eliza Hall Institute for Medical Research</a>,
Melbourne, Australia (<em>Nov 17</em>)</p>

<p>What is (are?) statistics?
<a href="https://www.biostat.wisc.edu/~kbroman/presentations/what_is_statistics.pdf"><img src="icons16/pdf-icon.png" alt="pdf" /></a>
<br />
<a href="http://research.marshfieldclinic.org/genetics/">Center for Medical Genetics</a>,
Marshfield Medical Research Foundation, Marshfield, Wisconsin (<em>Jan 8</em>)</p>

  </div>
</div>


      </div>

      <hr>

    </div>

    
  </body>
</html>

